packages/core/src/App/Containers/Layout/header/default-mobile-links.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, Icon, Popover } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { BinaryLink } from 'App/Components/Routes';
+import ShowNotifications from './show-notifications';
+import TradingHubOnboarding from './trading-hub-onboarding';
+
+export type TDefaultMobileLinks = {
+    handleClickCashier: () => void;
+};
```

Comment:
```suggestion
type TDefaultMobileLinks = {
    handleClickCashier: () => void;
};
```
The prop type of a component is for that component, Ideally, it shouldn't be used by anyone else so we shouldn't export it.